### PR TITLE
[l10n] Make automatic l10n depenpency work

### PR DESCRIPTION
**Pull request by henriksa**

### CODE REVIEW

File: rpm/sailfish-utilities.spec [hunk 1]
```diff
@@ -23,8 +23,9 @@ BuildRequires: qt5-default
 BuildRequires: qt5-qttools
 BuildRequires: qt5-qttools-linguist
 %if %{with l10n}
-%define _all_translations_version %(rpm -q --queryformat "%%{version}-%%{release}" %{name}-all-translations)
 BuildRequires: %{name}-all-translations
+#!BuildIgnore: %{name}-all-translations-pack
+%define _all_translations_version %(rpm -q --queryformat "%%{version}-%%{release}" %{name}-all-translations)
```
Comment (deztructor) [inline rpm/sailfish-utilities.spec]: Could you explain changes?

Comment (henriksa) [inline rpm/sailfish-utilities.spec]: sailfish-utilities-l10n-<lang> packages are provided by both sailfish-utilities-all-translations and sailfish-utilities-all-translation-pack. So to make the build work Ignore the pack one

File: rpm/sailfish-utilities.spec [hunk 2]
```diff
@@ -23,8 +23,9 @@ BuildRequires: qt5-default
 BuildRequires: qt5-qttools
 BuildRequires: qt5-qttools-linguist
 %if %{with l10n}
-%define _all_translations_version %(rpm -q --queryformat "%%{version}-%%{release}" %{name}-all-translations)
 BuildRequires: %{name}-all-translations
+#!BuildIgnore: %{name}-all-translations-pack
```
Comment (deztructor) [inline rpm/sailfish-utilities.spec]: And this is comment, isn't it?

File: rpm/sailfish-utilities.spec [hunk 3]
```diff
@@ -23,8 +23,9 @@ BuildRequires: qt5-default
 BuildRequires: qt5-qttools
 BuildRequires: qt5-qttools-linguist
 %if %{with l10n}
-%define _all_translations_version %(rpm -q --queryformat "%%{version}-%%{release}" %{name}-all-translations)
```
Comment (deztructor) [inline rpm/sailfish-utilities.spec]: I don not see visible changes in this line

**Pull request merged**